### ISSUE TITLE
`overrides.arrow` is deprecated in `type-annotation-spacing` since `@stylistic/eslint-plugin` v5.6.0

### ISSUE THREAD

**Issue by maiolica** [NishargShah/eslint-config-airbnb-extended#50]
### Steps to reproduce

I get this warning in the console since updating to `@stylistic/eslint-plugin` v5.6.0:

```console
[@stylistic/eslint-plugin]: You are using deprecated options("overrides.arrow") in "type-annotation-spacing", please use "arrow-spacing" instead.
```
see:
- https://github.com/NishargShah/eslint-config-airbnb-extended/blob/master/packages/eslint-config-airbnb-extended/rules/typescript/typescriptStylistic.ts#L132
- https://github.com/eslint-stylistic/eslint-stylistic/pull/1037

### Current behavior

I get a console warning about a deprecated syntax

### Expected behavior

Warn should be fixed

### Your environment

Desktop

- OS: `macOS Tahoe 26.1`
- Browser: `Chrome`
- EsLint Version: `9.39.1`
- Version: `2.3.2`

**Comment by NishargShah** [comment]
Issue resolved, please check

NPM: https://www.npmjs.com/package/eslint-config-airbnb-extended/v/3.0.0-beta-4
